src/functests/daltesthelpers.py

```diff
@@ -20,6 +20,8 @@ def init_user_dummy(user_id, password, interests):
     user_doc2 = struct.UserDocument.make_from_scratch(document=dummy_doc2, grade=0.5)
     dal.save_user_docs(new_user, [user_doc1, user_doc2])
 
+    return new_user
+
```

Comment:
if function return the user now, I would call init "create" instead of "init". I think "init" suggest a void function